src/models/Control.php

```diff
@@ -119,24 +119,24 @@ class Control extends Model {
   }
 
   public static async function importGame(): Awaitable<bool> {
-    $data_game = self::readJSON('game_file');
-    if ($data_game) {
-      $logos = must_have_idx($data_game, 'logos');
+    $data_game = JSONImporterController::readJSON('game_file');
+    if ($data_game instanceof KeyedContainer) {
```

Comment:
If the data isn't a KeyedContainer we should return `false` early. You can also `error_log` what the error was so we can find it in the logs later.